test/e2e/util/mongotester/mongotester.go

```diff
@@ -416,7 +416,7 @@ func getClientTLSConfig() (*tls.Config, error) {
 	caPool := x509.NewCertPool()
 	caPool.AppendCertsFromPEM(caPEM)
 
-	return &tls.Config{
+	return &tls.Config{ //nolint
```

Comment:
This is the same linter problem as in mongotester_test.go. I will file an issue to fix this separately.

Reply:
https://jira.mongodb.org/browse/CLOUDP-80410